src/server/game/Handlers/BattleGroundHandler.cpp

```diff
@@ -464,6 +464,8 @@ void WorldSession::HandleBattleFieldPortOpcode(WorldPacket& recvData)
                 _player->SetBattlegroundId(bg->GetInstanceID(), bg->GetBgTypeID(), queueSlot, true, bgTypeId == BATTLEGROUND_RB, teamId);
 
                 sBattlegroundMgr->SendToBattleground(_player, ginfo.IsInvitedToBGInstanceGUID, bgTypeId);
+
+                sScriptMgr->OnPlayerJoinedBattleground(_player);
```

Comment:
what about sending more info?? such as the BG type

Reply:
like?

Reply:
`bgTypeId` for example